build_system/docker_build.py

```diff
@@ -0,0 +1,77 @@
+
+import re
+import subprocess
+import sys
+
+from cross_build import BuildSystem
+import constants as c
+import build_utils as utils
+
+class DockerBuildSystem(BuildSystem):
+    def clean(self, config):
+        try:
+            self.exec_host_cmd("docker rm -f -v j2v8.$PLATFORM.$ARCH", config)
+        except subprocess.CalledProcessError:
+            return
+
+    def health_check(self, config):
+        try:
+            # general docker availability check
+            self.exec_host_cmd("docker stats --no-stream", config)
+
+            # NOTE: the additional newlines are important for the regex matching
+            version_str = utils.execute_to_str("docker version") + "\n\n"
```

Comment:
could you please try adding `, shell=True` to line 40:

`p = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True, cwd=cwd)`

I think that's what is missing there, once you can confirm the fix I will add it to the PP
Thanks